src/infrastructure/repositories/offre-emploi-http-sql.repository.ts

```diff
@@ -178,6 +178,11 @@ export interface OffreEmploiDto {
   }
   origineOffre: {
     urlOrigine: string
+    partenaires: [
+      {
+        url?: string
+      }?
```

Comment:
le point d'interrogation ici doit être mius après partenaires:

après le curly bracket ça veut dire que ton tableau peut contenir des undefined

Reply:
si je mets le ? après partenaire j'ai cette erreur "Type '[]' is not assignable to type '[{ url?: string | undefined; }]'.
  Source has 0 element(s) but target requires 1." c'est pour ça que je l'avais mis ici

Reply:
fait